docs/sources/features/datasources/loki.md

```diff
@@ -36,6 +36,21 @@ Just add it as a data source and you are ready to query your log data in [Explor
 | _URL_           | The URL of the Loki instance, e.g., `http://localhost:3100`                                                                                   |
 | _Maximum lines_ | Upper limit for number of log lines returned by Loki (default is 1000). Decrease if your browser is sluggish when displaying logs in Explore. |
 
+### Derived fields
+
+Derived fields configuration allow you to add additional fields parsed from the log message and optionally add link that uses the value of the field. This can be used for example to link to trace in your tracing backend from your logs or user profile page if userId is present int the log line. They are then showed in the [log details](/features/explore/#labels-and-parsed-fields)
```

Comment:
```suggestion
The Derived Fields configuration allows you to: 
* Add fields parsed from the log message. 
* Add a link that uses the value of the field. 

You can use this functionality to link to your tracing backend directly from your logs, or link to a user profile page if a userId is present in the log line. 

They are shown in the [log details](/features/explore/#labels-and-parsed-fields).
```

Reply:
Unclear antecedent. Is "They" in the last paragraph here the links, user ID, or something else? Please clarify.

Reply:
It refers to the links themselves so should probably be something like `These links will be shown in ...`

Reply:
Changed to `These links will be shown in`